### PR TITLE
feat: add gallery page

### PR DESCRIPTION
**Pull request by olehpratsko**

### CODE REVIEW

File: src/pages/gallery.jsx [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+                image={getImage(data.nfCoreHackathonImage9)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+          </div>
+          <div className="mt-8">
+            <Button to="https://photos.app.goo.gl/3BoRrqumcNqCShzf8" variant="accent" size="md" arrow>
+              See more in Google Photos
```
Comment (ewels) [inline src/pages/gallery.jsx:332]: Can we add the exact number of photos in each gallery? Helps to make it clear that each button is different, and how many there are to see..

```suggestion
              See XXX more images in Google Photos
```

File: src/pages/gallery.jsx [hunk 2]
```diff
@@ -0,0 +1,529 @@
...
+
+  return (
+    <>
+      <Seo
+        title="Nextflow Events Gallery"
+      />
+      <div className="py-32 bg-gray-900 text-white">
+        <div className="container-lg text-center">
+          <h1 className="typo-h2 mb-4">
+            Nextflow Events Gallery
```
Comment (ewels) [inline src/pages/gallery.jsx:236]:
```suggestion
            Nextflow Summit Photos
```

File: src/pages/gallery.jsx [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+          </h1>
+          <p className="typo-body max-w-md mb-4 mx-auto">
+            Find photo reports from our latests events in this section
```
Comment (ewels) [inline src/pages/gallery.jsx:239]:
```suggestion
            See photos from the nf-core hackathon and Nextflow Summit 2022.
```

**Review by ewels — COMMENTED** [review]
Images need to be resized, they are massive and take a long time to load.

Otherwise looking great except the comments I made 👍🏻

File: src/pages/gallery.jsx [hunk 4]
```diff
@@ -0,0 +1,529 @@
...
+                image={getImage(data.nfCoreHackathonImage9)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+          </div>
+          <div className="mt-8">
+            <Button to="https://photos.app.goo.gl/3BoRrqumcNqCShzf8" variant="accent" size="md" arrow>
+              See more in Google Photos
```
Comment (mribeirodantas) [inline src/pages/gallery.jsx:332]:
```suggestion
              See 174 more images in Google Photos
```

File: src/pages/gallery.jsx [hunk 5]
```diff
@@ -0,0 +1,529 @@
...
+                image={getImage(data.summitDinnerImage10)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="Nextflow Summit Social Dinner at Shoko"
+              />
+            </div>
+          </div>
+          <div className="mt-8">
+            <Button to="https://photos.app.goo.gl/Hp5Ybf3YUFT5PQA49" variant="accent" size="md" arrow>
+              See more in Google Photos
```
Comment (mribeirodantas) [inline src/pages/gallery.jsx:426]:
```suggestion
              See 58 more images in Google Photos
```

File: src/pages/gallery.jsx [hunk 6]
```diff
@@ -0,0 +1,529 @@
...
+                image={getImage(data.summitImage10)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="Nextflow Summit"
+              />
+            </div>
+          </div>
+          <div className="mt-8">
+            <Button to="https://photos.app.goo.gl/82mXfCrx3ukMy2zm9" variant="accent" size="md" arrow>
+              See more in Google Photos
```
Comment (mribeirodantas) [inline src/pages/gallery.jsx:520]:
```suggestion
              See 115 more images in Google Photos
```

**Review by mribeirodantas — APPROVED** [review]

**Pull request merged**